src/phoenix/experimental/callbacks/langchain_tracer.py

```diff
@@ -13,25 +22,50 @@ def _langchain_run_type_to_span_kind(run_type: str) -> SpanKind:
     return SpanKind(run_type.upper())
 
 
+def _serialize_json(obj: Any) -> str:
+    if isinstance(obj, datetime):
+        return obj.isoformat()
+    return str(obj)
+
+
+def _convert_io(io: Optional[Dict[str, Any]]) -> Iterator[Any]:
+    if io is None:
+        return
+    if len(io) == 1 and isinstance(value := next(iter(io.values())), str):
+        yield value
```

Comment:
Did you mean to also `yield MimeType.TEXT` here? If so, I think you can change the functions return type to `Iterator[Tuple[Any, MimeType]]`

Reply:
no, because by default (i.e. None) it's assumed to be TEXT